build.py

```diff
@@ -545,12 +563,19 @@ def create_dockerfile_buildbase(ddir, dockerfile_name, argmap, backends):
 RUN rm -fr *
 COPY . .
 ENTRYPOINT []
+'''
+        df += install_dcgm_libraries()
+        df += fail_if_dcgm_api_export_exists()
+        df += '''
+WORKDIR /workspace
```

Comment:
I believe this line is not needed. The WORKDIR is already set to /workspace earlier